### PR TITLE
[docs] add descriptions for new examples

### PR DESCRIPTION
**Pull request by anthophobiac**

### CODE REVIEW

File: examples/iterate-over-table-rows/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Iterate Over Table Rows
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to iterate over table rows. 
+
+The [example page](index.html) provides a table, which we iterate over. First off, we need to select the table and then get the total number of rows in it:
+
+`const dataRowCount = await dataRows.count;`
```
Comment (VasilyStrelyaev) [inline examples/iterate-over-table-rows/README.md]: It looks better if you use code blocks

File: examples/iterate-over-table-rows/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Iterate Over Table Rows
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to iterate over table rows. 
+
+The [example page](index.html) provides a table, which we iterate over. First off, we need to select the table and then get the total number of rows in it:
```
Comment (VasilyStrelyaev) [inline examples/iterate-over-table-rows/README.md]: I wouldn't say that the page *provides* a table. BTW, Ray suggested that we minimize the usage of 'provide'. The page *contains* a table simply.

Comment (VasilyStrelyaev) [reply examples/iterate-over-table-rows/README.md]: Then, it's better to use the third person in docs. Not *we iterate* but *the example iterates*, *test code iterates*, *the function returns*, etc.

Comment (VasilyStrelyaev) [reply examples/iterate-over-table-rows/README.md]: Finally, a preposition at the end of the sentence *...iterate over*.

Comment (VasilyStrelyaev) [reply examples/iterate-over-table-rows/README.md]: So, probably what you need is:

*The [example page](index.html) contains a table over which the test code iterates.*

(or something better :smile:)

File: examples/iterate-over-table-rows/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+`const dataRowCount = await dataRows.count;`
+
+When you have `dataRowCount` available, you can proceed to iterate over the rows. In the suggested example, we do this in a `for` loop.
```
Comment (VasilyStrelyaev) [inline examples/iterate-over-table-rows/README.md]: No need for:
* *you have ... available* - just *... is available*
* *proceed to*
* *suggested*

File: examples/iterate-over-table-rows/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+This example shows how to iterate over table rows. 
+
+The [example page](index.html) provides a table, which we iterate over. First off, we need to select the table and then get the total number of rows in it:
+
+`const dataRowCount = await dataRows.count;`
+
+When you have `dataRowCount` available, you can proceed to iterate over the rows. In the suggested example, we do this in a `for` loop. 
+
+In each row, we check the cell where the index is equal to 1 and expect it to contain the text `Smith`. If the following row is even, we expect it to have a checkbox in a checked state.
```
Comment (VasilyStrelyaev) [inline examples/iterate-over-table-rows/README.md]: To me,  saying *expects smth to* or *expects that smth* about tests (as in *the test expects it to...*) is okay, but the correctors will change it to *the test checks that...*

File: examples/wait-for-file-download/README.md [hunk 5]
```diff
@@ -1 +1,11 @@
-# Wait for file download
+# Wait For File Download
+
+**Test Code**: [test.js](test.js)
+
+This example shows how to set a test to wait for the file download.
+
+Before the test begins, we need to obtain the `downloadedFilePath`. If the file already exists on the path, we remove the existing one to make way for the download.
```
Comment (VasilyStrelyaev) [inline examples/wait-for-file-download/README.md]: *make way for* - awesome for a blog, but we'd better use something more neutral in tech docs, like *the test removes it to save the downloaded file without warnings/errors* (just an idea)

File: examples/wait-for-file-download/README.md [hunk 6]
```diff
@@ -1 +1,11 @@
...
+
+In the provided example, we suggest running the test using Chrome to simplify the process of locating the downloaded file. If the browser is Chrome, we initiate the asynchronous `waitForDownload` function.
+
+`waitForDownload` runs a `for` loop which checks if the file exists on the specified download path. It runs the body of the loop 11 times, with a 500-millisecond timeout between tries, untill the file is downloaded. If the function does not locate the file after the tries, it returns `false`.
```
Comment (VasilyStrelyaev) [inline examples/wait-for-file-download/README.md]: typo in *until*

Comment (VasilyStrelyaev) [reply examples/wait-for-file-download/README.md]: I don't think it makes sense to specify the exact numbers in this description - they are too hard to read and, in fact, they are not essential.

File: examples/wait-for-file-download/README.md [hunk 7]
```diff
@@ -1 +1,11 @@
-# Wait for file download
+# Wait For File Download
+
+**Test Code**: [test.js](test.js)
+
+This example shows how to set a test to wait for the file download.
+
+Before the test begins, we need to obtain the `downloadedFilePath`. If the file already exists on the path, we remove the existing one to make way for the download.
+
+In the provided example, we suggest running the test using Chrome to simplify the process of locating the downloaded file. If the browser is Chrome, we initiate the asynchronous `waitForDownload` function.
```
Comment (VasilyStrelyaev) [inline examples/wait-for-file-download/README.md]: many -ing's - *we suggest that*, *to make it easier to locate*.
Besides, does using Chrome actually simplifies locating the file?

Comment (anthophobiac) [reply examples/wait-for-file-download/README.md]: @VasilyStrelyaev it says so in the code itself, in a comment:

```
 // Run this test only with the Google Chrome browser to simplify the search for the downloaded file.
        if (t.browser.name !== 'Chrome')
            return;
```

**Pull request closed without merging**